game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleState.java

```diff
@@ -0,0 +1,12 @@
+package games.strategy.triplea.delegate.battle;
+
+import games.strategy.engine.data.Unit;
+import java.util.Collection;
+
+/** Exposes the battle state and allows updates to it */
```

Comment:
I plan to add those.  My plan is to add the existing methods from `MustFightBattle` that are needed in the steps.  As I convert each step, I'll be adding more and more methods to the two interfaces.  Some of these methods will be mutator methods.  I don't plan on renaming them at that point, though.

Then, I plan on splitting up methods that have functionality that should be in `BattleActions` (IDelegateBridge interaction) and `BattleState`.  An example of this is `MustFightBattle#queryRetreat`.

Once I have the methods split between the two interfaces, I'll look into refactoring them so that they have a targeted focus.